vespa-http-client/src/main/java/com/yahoo/vespa/http/client/core/communication/IOThread.java

```diff
@@ -319,12 +319,12 @@ private ThreadState cycle(final ThreadState threadState) {
                     successfullHandshakes.getAndIncrement();
                 } catch (ServerResponseException ser) {
                     executeProblemsCounter.incrementAndGet();
-                    log.log(Level.INFO, "Handshake did not work out " + endpoint, Exceptions.toMessageString(ser));
+                    log.info("Handshake did not work out " + endpoint + ": " + Exceptions.toMessageString(ser));
```

Comment:
Alternatively (because I want a blue bike shed!), `log.info(() -> String.format("Handshake did not work out %s: %s", endpoint, Exceptions.toMessageString(ser)));`